### PR TITLE
Add Qualifyze logo to discover page

### PR DESCRIPTION
**Pull request by 0xdfgp**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Docs
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

## Other information
Qualifyze is delighted to use Nest as the official framework for the development of our platform. Thanks!

### CODE REVIEW

File: src/assets/logo/qualifyze.svg
Comment (jonatannietoa-qualifyze) [inline src/assets/logo/qualifyze.svg:1]: @0xdfgp crop logo

Comment (jonatannietoa-qualifyze) [reply src/assets/logo/qualifyze.svg:1]: Add URL in line 175 in content/discover/who-uses.json

**Comment by 0xdfgp** [comment]
> For now, we decided to suspend adding more logos as there's a decent number of logos present there already and the page itself is getting bloated with images (we may even remove some of them in the near future). Instead, please consider adding yourself to the list below!

Hey Kamil, firstly, I would like to express my sincere appreciation for the exceptional framework you have created. Your work is truly commendable.
Regarding the logo, I noticed a duplication of the IBM logo. In light of this, I just pushed a commit replacing the second instance with our company's logo. However, I completely understand if anyway you prefer not to add it at this time.
Please review this change and let me know if it aligns with your preferences. If it doesn't, I am more than willing to revert the logo and maintain the URL at the bottom of the list. 

Thanks again!

**Comment by kamilmysliwiec** [comment]
We rejected so many different logos in the past (for the exact same reason) that I honestly wouldn't feel comfortable accepting one now 😞

**Pull request closed without merging**